src/cr/cube/matrix.py

```diff
@@ -2378,7 +2378,7 @@ def base(self):
         a 1D np.int64 ndarray. A vector opposing a CAT dimension has a single base and
         produces a scalar np.int64 value.
         """
-        return np.sum(self._unweighted_counts)
+        return np.sum(self._unweighted_counts, axis=0)
```

Comment:
This is unusual, I wasn't expecting this... Can you give some more context to this? Do we need to update the docstring? It might be that I should review exactly what this does, and that lack of memory is the only reason for this comment, but still :)

Reply:
Okay, this might have been an extraction too far, but I documented this like so:

```python
        # --- The `axis=0` parameter makes this work for both the X_MR and X_CAT cases.
        # --- A vector opposing an MR dimension (X_MR) has a 2D unweighted_counts like:
        #
        #      [[653  128  120  389  469]
        #       [101   95   96   69   61]]
        #
        # --- where the first axis is selected/unselected and the second dimension is
        # --- a distinct value for each MR-subvar. The sum must include both selected
        # --- and unselected values, but cannot sum across subvars. So the result is
        # --- a 1D array for the X_MR case.
        # ---
        # --- In the X_CAT case, unweighted_counts is a 1D array like:
        #
        #      [246  61  102  318  469]
        #
        # --- Summing across axis-0 gives the right (scalar) result here too, even
        # --- though specifying the axis is superfluous for this case.
```